tests/test_DisentangledBetaVAE.py

```diff
@@ -0,0 +1,702 @@
+import os
+from copy import deepcopy
+
+import pytest
+import torch
+from torch.optim import SGD, Adadelta, Adagrad, Adam, RMSprop
+
+from pythae.customexception import BadInheritanceError
+from pythae.models.base.base_utils import ModelOutput
+from pythae.models import DisentangledDisentangledBetaVAE, DisentangledBetaVAEConfig
```

Comment:
Issue in model import

Reply:
Even when this import issue is solved, I still get these errors when running the test on `DisentangledBetaVAE`. It seems that the number of epoch is not updated correctly in the model. 


```bash
============================================================================================= short test summary info =============================================================================================
FAILED test_DisentangledBetaVAE.py::Test_DisentangledBetaVAE_Training::test_betavae_main_train_loop[betavae0-training_configs0-Adam-model_configs0] - assert 3 == 1
FAILED test_DisentangledBetaVAE.py::Test_DisentangledBetaVAE_Training::test_betavae_main_train_loop[betavae0-training_configs0-Adam-model_configs1] - assert 3 == 1
FAILED test_DisentangledBetaVAE.py::Test_DisentangledBetaVAE_Training::test_betavae_main_train_loop[betavae1-training_configs0-Adam-model_configs0] - assert 3 == 1
FAILED test_DisentangledBetaVAE.py::Test_DisentangledBetaVAE_Training::test_betavae_main_train_loop[betavae1-training_configs0-Adam-model_configs1] - assert 3 == 1
FAILED test_DisentangledBetaVAE.py::Test_DisentangledBetaVAE_Training::test_betavae_main_train_loop[betavae2-training_configs0-Adam-model_configs0] - assert 3 == 1
FAILED test_DisentangledBetaVAE.py::Test_DisentangledBetaVAE_Training::test_betavae_main_train_loop[betavae2-training_configs0-Adam-model_configs1] - assert 3 == 1
FAILED test_DisentangledBetaVAE.py::Test_DisentangledBetaVAE_Training::test_betavae_main_train_loop[betavae3-training_configs0-Adam-model_configs0] - assert 3 == 1
FAILED test_DisentangledBetaVAE.py::Test_DisentangledBetaVAE_Training::test_betavae_main_train_loop[betavae3-training_configs0-Adam-model_configs1] - assert 3 == 1
FAILED test_DisentangledBetaVAE.py::Test_DisentangledBetaVAE_Training::test_betavae_main_train_loop[betavae4-training_configs0-Adam-model_configs0] - assert 3 == 1
FAILED test_DisentangledBetaVAE.py::Test_DisentangledBetaVAE_Training::test_betavae_main_train_loop[betavae4-training_configs0-Adam-model_configs1] - assert 3 == 1
============================================================================ 10 failed, 3013 passed, 121 warnings in 218.15s (0:03:38) ============================================================================

```